.github/workflows/es-spellcheck.yml

```
@@ -0,0 +1,30 @@
+---
+# SPDX-license-identifier: Apache-2.0
+##############################################################################
+# Copyright (c) 2022
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+##############################################################################
+
+name: Spellcheck CI
+# yamllint disable-line rule:truthy
+on:
+  pull_request:
+    paths:
+      - 'content/es/*.md'
```

If this line does not work, then we might try this:
```suggestion
      - 'content/es/**.md'
```